src/main/java/domain/car/Car.java

```diff
@@ -0,0 +1,38 @@
+package domain.car;
+
+import domain.name.Name;
+
+public class Car {
+    private static final int INITIAL_POSITION=0;
+    public static final int FORWARD_MAX_NUMBER = 10;
+    private final Name name;
+    private int position;
+    private static final int FORWARD_THRESHOLD = 4;
+
+    public Car(Name name) {
+        this.name = name;
+        this.position = INITIAL_POSITION;
+    }
+
+    public Name getName() {
+        return name;
+    }
+
+    public Integer getPosition() {
+        return position;
+    }
+
+    public void race(Integer number) {
+        if (number >= FORWARD_THRESHOLD) {
+            this.position++;
+        }
+    }
+
+    public boolean comparePosition(int position){
+        if(this.position==position){
+            return true;
+        }
+        return false;
```

Comment:
```suggestion
        return this.position == position;
```
으로 바로 표현할 수 있겠네요.

코딩 컨벤션도 안맞춰져있는데 항상 코드 포맷팅을 실행시켜보는 것을 습관화해보는 것이 중요합니다 :)

Reply:
명심하겠습니다😀